### PR TITLE
Move MetaPhlAn3 to a subcommand and add mergemetaphlantables

### LINKED ISSUE #2024: new module: METAPHLAN3/MERGEMETAPHLANTABLES
### Is there an existing module for this?

- [X] I have searched for the existing module

### Is there an open PR for this?

- [X] I have searched for existing PRs

### Is there an open issue for this?

- [X] I have searched for existing issues

### Are you going to work on this?

- [X] If I'm planning to work on this module, I added myself to the `Assignees` to facilitate tracking who is working on the module

### PR DESCRIPTION
**Pull request by jfy133**
Closes #2024 

What a saga...

## PR checklist

Closes #XXX 

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [ ] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [ ] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: tests/modules/metaphlan3/mergemetaphlantables/test.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+- name: metaphlan3 mergemetaphlantables test_metaphlan3_mergemetaphlantables
+  command: nextflow run ./tests/modules/metaphlan3/mergemetaphlantables -entry test_metaphlan3_mergemetaphlantables -c ./tests/config/nextflow.config  -c ./tests/modules/metaphlan3/mergemetaphlantables/nextflow.config
+  tags:
+    - metaphlan3
+    - metaphlan3/mergemetaphlantables
+  files:
+    - path: output/metaphlan3/merged_abundance_table.txt
+      md5sum: 9890cde773a3b197f1693fbea2de38e1
```
Comment (jfy133) [inline tests/modules/metaphlan3/mergemetaphlantables/test.yml:8]:
```suggestion
      contains: ["clade_name	NCBI_tax_id	"]
```

File: modules/metaphlan3/mergemetaphlantables/main.nf [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    prefix = task.ext.prefix ?: "merged_abundance_table"
+    """
+    merge_metaphlan_tables.py \\
+        $args \\
+        -o ${prefix}.txt \\
```
Comment (FriederikeHanssen) [inline modules/metaphlan3/mergemetaphlantables/main.nf:25]: if no multithreading, would the `process_single` label be enough?

File: modules/metaphlan3/metaphlan3/main.nf [hunk 3]
```diff
@@ -9,7 +9,7 @@ process METAPHLAN3 {
 
     input:
     tuple val(meta), path(input)
-    path metaphlan_db
+    path metaphlan_bt2_db
```
Comment (FriederikeHanssen) [inline modules/metaphlan3/metaphlan3/main.nf:12]: the yml probably needs some renaming for this one

**Review by FriederikeHanssen — APPROVED** [review]
looks good 🥳 , just the label question

File: modules/metaphlan3/mergemetaphlantables/main.nf (hunk 2 above)
Comment (jfy133) [reply modules/metaphlan3/mergemetaphlantables/main.nf:25]: Yes! I keep forgetting about this, I just run through the default on the dialogue :facepalm:

File: modules/metaphlan3/mergemetaphlantables/main.nf [hunk 4]
```diff
@@ -0,0 +1,33 @@
+process METAPHLAN3_MERGEMETAPHLANTABLES {
+    label 'process_low'
```
Comment (jfy133) [inline modules/metaphlan3/mergemetaphlantables/main.nf:2]:
```suggestion
    label 'process_single'
```

File: modules/metaphlan3/metaphlan3/main.nf (hunk 3 above)
Comment (jfy133) [reply modules/metaphlan3/metaphlan3/main.nf]: Wasn't even in hte YAML in the first place :scream:

**Pull request merged**